website/website/assets/js/map-en.js

```diff
@@ -556,6 +556,7 @@ function httpPostAsync(endPoint, params, callback) {
 	}
 	console.log(theUrl)
 	xmlHttp.open("POST", theUrl, true); // true for asynchronous
+    xmlHttp.setRequestHeader('Content-Type', 'application/json');
```

Comment:
@matthiasschaub your indention is wrong here

Reply:
Fixed